synthtool/gcp/templates/java_library/.kokoro/release/publish_javadoc.sh

```diff
@@ -59,6 +59,9 @@ python3 -m docuploader upload . \
 
 popd
 
+# V2 currently requires Java 8 ONLY.
```

Comment:
Could you check for Java 8 instead of printing the version?

Reply:
Why?  It's just there to remind someone when they run across a problem.

Reply:
PTAL